sriov-network-device-plugin.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: sriov-network-device-plugin
+  version: 3.8.0
+  epoch: 0
+  description: SRIOV network device plugin for Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: a380ca568b4f85e105255af0aa7facc0128d611e
+      repository: https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: github.com/opencontainers/runc@v1.1.14
+
+  - uses: go/build
+    with:
+      output: sriovdp
+      packages: ./cmd/sriovdp
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}/"
+      install -Dm755 ./images/entrypoint.sh "${{targets.contextdir}}/"
```

Comment:
This would likely be better to have in an -entryooint subpackage if the plugin is useful elsewhere